db/db.go

```diff
@@ -3,20 +3,48 @@ package db
 import (
 	"database/sql"
 	"fmt"
+	"os"
 
+	"github.com/qwsnxnjene/go_final_project/tests"
 	_ "modernc.org/sqlite"
 )
 
+var ActualDbPath string
+
+// CreateDB() получает путь к файлу с базой данных и, если необходимо, создаёт таблицу
+func CreateDB() {
+	dbPath := tests.DBFile
+	if path := os.Getenv("TODO_DBFILE"); path != "" {
+		dbPath = path
+	}
+	_, err := os.Stat(dbPath)
+
+	ActualDbPath = dbPath
+
+	// if there is no DB, create one
+	var install bool
+	if err != nil {
+		install = true
+	}
+	if install {
+		CreateTable(dbPath)
+	}
+}
+
+// CreateTable(path) создаёт таблицу в файле, указанном в path
+// возвращает nil при успешном создании таблицы, иначе ошибку
 func CreateTable(path string) error {
 	db, err := sql.Open("sqlite", path)
+
 	if err != nil {
+		fmt.Println(err.Error())
 		return fmt.Errorf("[db.CreateTable]: can't open database: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (`wrap`)
```suggestion
		return fmt.Errorf("[db.CreateTable]: can't open database: %w", err)
```